Makefile.include

```diff
@@ -72,6 +72,9 @@ include $(RIOTMAKE)/docker.inc.mk
 # include color echo macros
 include $(RIOTMAKE)/color.inc.mk
 
+# include concurrency helpers
+include $(RIOTMAKE)/info-nproc.inc.mk
```

Comment:
This file is missing. + why not define it in vars.inc.mk?

Reply:
ups.

> why not define it in vars.inc.mk?

that file only contains the variables to be exported. NPROC has like 20 lines, IMO it should be factored out, anyways.

Reply:
> that file only contains the variables to be exported. NPROC has like 20 lines, IMO it should be factored out, anyways.

Ah sorry, I thought it was just the assignment of `NPROC`.